### ISSUE TITLE
Final review

### ISSUE THREAD

**Issue by MasonGallo** [blacki/congress#3]
Clarity of research problem:

I think your research problem is clear. I didn't see any slides at time of review, so I reviewed the readme. The topic modeling relevance section is very insightful and should definitely be one of the first sections to introduce the problem to the reader.

Code:

I would like to see some development of how "good" your models are. One possible way to do this is to do a test/train split and compare some sort of distance metric on the test set based on varying the hyperparameters aka # topics, etc on the train set.

Next steps:

Put together a clean pdf to present your work, findings, and conclusions. I would dedicate a section to evaluating how "good" your topic modeling is, as discussed above.
